src/locales/en.json

```diff
@@ -143,7 +143,12 @@
     },
     "prevalence_info_source_information": "Prevalence data consolidated from <1>Johns Hopkins CSSE</1> (reported cases), <4>Covid Act Now</4> (US positive test rates), <7>Our World in Data</7> (international positive test rates), and <10>Coronavirus COVID-19 România</10> (Romania reported cases).",
     "select_scenario": "Start with a premade scenario or <2>build your own</2>:",
-    "no_prebuilt_scenario_found": "We don't have this scenario yet, but you can build your own below.",
+    "no_prebuilt_scenario_found": "No premade scenario found, but you may",
+    "saved_scenario_continue_custom": "Build your own scenario",
+    "scenario_show_results": "Showing results for",
+    "scenario_guide_through": "Guiding you through",
+    "select_scenario_start_over": "Restart",
```

Comment:
Could this be "Choose a different scenario"?

Reply:
Done

Reply:
I'd prefer a shorter string here that people can pattern match from other contexts without having to fully read the prompt I think.